packages/calling/src/CallingClient/line/line.test.ts

```diff
@@ -154,4 +157,104 @@ describe('Line Tests', () => {
       expect(line.getRegistrationStatus()).toEqual(MobiusStatus.DEFAULT);
     });
   });
+  describe('Line calling tests', () => {
+    let line;
+
+    beforeEach(() => {
+      line = new Line(
+        userId,
+        clientDeviceUri,
+        LineStatus.ACTIVE,
+        mutex,
+        primaryMobiusUris(),
+        backupMobiusUris(),
+        LOGGER.INFO
+      );
+    });
+
+    afterEach(() => {
+      jest.clearAllTimers();
+      jest.clearAllMocks();
+      jest.useRealTimers();
+      line.removeAllListeners();
+    });
+    it('Return a successful call object while making call', () => {
```

Comment:
For successful cases, we should have a check for createCallSpy getting invoked with rights et of arguments. Make use of toBeCalledOnceWith

Reply:
Added check for createCall